exchange/exchange.go

```diff
@@ -114,6 +126,28 @@ func (e *exchange) buildBidResponse(liveAdapters []openrtb_ext.BidderName, adapt
 		bidResponse.NBR = openrtb.NoBidReasonCode.Ptr(openrtb.NoBidReasonCodeInvalidRequest)
 	}
 
+	// Process the request to check for targeting parameters.
+	targData := &targetData{
+		targetFlag:	false,
+		lengthMax:	0,
+		priceGranularity:	openrtb_ext.PriceGranularityMedium,
+		cpm:		0.0,
+		bid:		make(map[string]*openrtb.Bid, len(bidRequest.Imp)),
+		bidder:		make(map[string]openrtb_ext.BidderName, len(bidRequest.Imp)),
+	}
+	requestExt := new(openrtb_ext.ExtRequest)
+	err := json.Unmarshal(bidRequest.Ext, requestExt)
+	if err != nil {
+		errList = append(errList, fmt.Errorf("Error decoding Request.ext : %s", err.Error()))
+	}
+	if requestExt.Prebid.Targeting != nil {
+		if len(requestExt.Prebid.Targeting.PriceGranularity) > 0 {
+			targData.targetFlag = true
```

Comment:
Is `targData.targetFlag` only true if there is a defined `PriceGranularity` on the requestExt.Prebid.Targeting? 

Is this new logic? Before ORTB, we pull in price granularity from account settings. Does this logic require that we pass that parameter from client side code?

Reply:
At the moment, yes. Probably need to understand how we identify account better. Currently (from another PR) a stored ad unit config will be inserted into the request before exchange sees it. We should do the same for account, rather than having two potential locations for this data when we reach this stage.

Reply:
Yeah if we pull in PriceGranularity from account before we get to this point, then the logic makes sense to me. I just wanted to make sure that its not a mobile requirement to pass in `PriceGranularity` from the client side in order to have `targData.targetFlag` set to true.

Reply:
yeah. There's actually see some related code up for review in #207. That PR only supports server-side request data _by id_. In a future PR, we'll extend support for request data _by account_.

All that merging will happen during input parsing, though, before it gets to the exchange.